debian/control

```diff
@@ -0,0 +1,18 @@
+Source: fortio
+Section: web
+Priority: optional
+Maintainer: Laurent Demailly <ldemailly@gmail.com>
+Build-Depends: debhelper (>= 10),
+	golang-go (>= 1.8)
```

Comment:
I think `lintian (>=2.5)` should be added here.

Reply:
thanks for checking! lintian is only needed for the extra test makefile target (but so are build essentials, dh*, golang etc...) not for the build itself